front/src/applications/stdcmV2/types.ts

```diff
@@ -30,3 +30,19 @@ export type StdcmConfigCardProps = {
   setCurrentSimulationInputs: React.Dispatch<React.SetStateAction<StdcmSimulationInputs>>;
   disabled?: boolean;
 };
+
+export enum ArrivalTimeTypes {
+  PRECISE_TIME = 'preciseTime',
+  ASAP = 'asSoonAsPossible',
+}
+
+export enum StdcmErrorTypes {
```

Comment:
I think `StdcmConfigErrorTypes` would be a better name because later we will also have to manage the response errors
```suggestion
export enum StdcmConfigErrorTypes {
```